sunbeam-python/sunbeam/versions.py

```diff
@@ -161,6 +163,15 @@
         }
     }
 }
+DEPLOY_K8S_TFVAR_MAP = {
+    "charms": {
+        "k8s": {
+            "channel": "k8s_channel",
```

Comment:
k8s and microk8s vars don't match, which could be confusing. and we tend to use hyphens more